### PR TITLE
TreeBuilder Symfony 5.x constructor update

### PR DESCRIPTION
**Pull request by palazzetti**
New constructor at https://github.com/symfony/symfony/blob/8361713367e2735fd9b63cd2b6661411a0339811/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php#L26

### CODE REVIEW

**Comment by BlackbitDevs** [comment]
Can we find a solution which is also compatible with https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php?

**Comment by palazzetti** [comment]
> Can we find a solution which is also compatible with https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php?

Done

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -29,8 +30,15 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('alep_ldap');
+        $kernelVersion = Kernel::MAJOR_VERSION;
+
+        if ($kernelVersion > 5) {
```
Comment (BlackbitDevs) [inline src/DependencyInjection/Configuration.php]: \> 5 or >= 5?

**Review by BlackbitDevs — CHANGES_REQUESTED** [review]

File: src/DependencyInjection/Configuration.php (hunk 1 above)
Comment (palazzetti) [reply src/DependencyInjection/Configuration.php]: You are right, >= 5

**Comment by BlackbitDevs** [comment]
Thanks a lot! Are you yet using this bundle with Pimcore 10?

**Comment by palazzetti** [comment]
> Thanks a lot! Are you yet using this bundle with Pimcore 10?

Yes and I have no problems, good project 👍

**Pull request merged**